controllers/managedocs_controller.go

```diff
@@ -91,6 +92,7 @@ const (
 	openshiftMonitoringNamespace           = "openshift-monitoring"
 	alertRelabelConfigSecretName           = "managed-ocs-alert-relabel-config-secret"
 	alertRelabelConfigSecretKey            = "alertrelabelconfig.yaml"
+	onboardingTicketSecretName             = "onboarding-ticket-key"
```

Comment:
Is this the name the product expects for the secret? even if it is let's name the var something more sane.
Maybe: `onboardingValidationKeySecretName`